src/core/Akka.Cluster.Tests/StartupWithOneThreadSpec.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Threading;
+using Akka.Actor;
+using Akka.Actor.Dsl;
+using Akka.Configuration;
+using Akka.Event;
+using Akka.TestKit;
+using Akka.Util;
+using Xunit;
+
+namespace Akka.Cluster.Tests
+{
+    public class StartupWithOneThreadSpec : AkkaSpec
```

Comment:
Spec designed to reproduce deadlock failures using a very small dedicated thread pool.